### PR TITLE
[flash_ctrl] flash_ctrl countermeasure list and annotation

### PR DESCRIPTION
**Pull request by tjaychen**
This is gated by #10626

### CODE REVIEW

**Comment by sriyerg** [comment]
Hey @tjaychen, PR #10556 has merged, which auto-generates a testplan from the listed security countermeasures. Please rebase, run `make -C hw`, make manual fixes as need and re-push. Thanks!

**Comment by tjaychen** [comment]
will do. Let me do that once others have had a chance to look at the changes here, so I can address all the comments / changes in one shot.

**Review by eunchan — APPROVED** [review]

File: hw/ip/flash_ctrl/data/flash_ctrl.hjson [hunk 1]
```diff
@@ -177,14 +177,60 @@
...
+      desc: "Software control for flash enable is 32-bit constant."
+    }
+    { name: "MEM.SCRAMBLE",
+      desc: "Memory is XEX scrambled."
+    }
+    { name: "MEM.INTEGRITY",
+      desc: "Memory is protected with two layers of ECC integrity."
+    }
+    // This name, if approved, needs to be added to the spec
+    // { name: "RMA_ENTRY.MEM.WIPE",
```
Comment (msfschaffner) [inline hw/ip/flash_ctrl/data/flash_ctrl.hjson:223]: I believe we have just recently added `SEC_WIPE` as a valid countermeasure name for something inside AES, so you could use that one...
https://github.com/lowRISC/opentitan/blob/cdc4bd4c821fce7a32367ee7450358d0f9a7f0d2/util/reggen/countermeasure.py#L55

**Review by msfschaffner — APPROVED** [review]
LGTM

**Pull request merged**